python-avd/pyavd/_eos_designs/structured_config/underlay/ethernet_interfaces.py

```diff
@@ -67,7 +67,7 @@ def ethernet_interfaces(self: AvdStructuredConfigUnderlay) -> list | None:
                 )
 
                 # PTP
-                if get(link, "ptp.enable") is True:
+                if get(link, "ptp.enable") is True and link["interface"] in self.shared_utils.ptp_uplinks:
```

Comment:
This is too late. You should enforce this in the eos_designs_facts under the `uplinks` like we talked about.

Reply:
That is the `uplink_ptp` which is the old key which we might have to deprecate soon. 
@ClausHolbechArista